processor/src/operations/io_ops.rs

```diff
@@ -72,9 +78,17 @@ impl Process {
     ///   removed from the stack.
     ///
     /// Thus, the net result of the operation is that the stack is shifted left by one item.
+    ///
+    /// Helper registers are filled with values that were stored in memory before the operation.
```

Comment:
nit: maybe specify how many registers were filled. For example: "The first 4 helper registers are filled with the value which were stored in memory before the operation."